### DISCUSSION TITLE
When resuming with BackboneFinetuning callback, backbone parameters has requires_grad=False

### DISCUSSION THREAD

**Discussion by sour4bh** [Lightning-AI/pytorch-lightning#9703] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
When I resume from a checkpoint that has the backbone unfrozen (i.e. current_epoch > unfreeze_backbone_at_epoch), the parameters of the backbone are once again frozen. 
I checked this in the following manner:
```python
def training_epoch_end(self, _):
    for name, param in self.named_parameters():
        print(param.requires_grad, name)
```
Although I was able to verify that the optimizer upon resuming from checkpoint does contain two param groups which is expected.

**Comment by rohitgr7** [comment]
can you share a minimal example to reproduce. will help debug this
